ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -675,9 +675,9 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "ID": "messageJudgeOrLegalAdviser",
```

Comment:
Should I also change the ID for this?

Reply:
If you want to note that you will have to update any authorisation entry, any case event to field entry, and any case event to complex type entry that uses it. It might also be used in any labels that are links to start the event.

Reply:
The task only seems to be asking to rename it in the dropdown and no functionality has changed so I think it is probably fine to leave it with the current ID as it still describes the event

Reply:
seems sensible, would be a lot more additional work otherwise